python/src/iceberg/utils/datetime.py

```diff
@@ -36,6 +39,15 @@ def micros_to_days(timestamp: int) -> int:
     return (datetime.fromtimestamp(timestamp / 1_000_000) - EPOCH_TIMESTAMP).days
 
 
+def micros_to_time(micros: int) -> time:
```

Comment:
Jun just added `time_from_micros` below that is basically the same thing (though I like this version better because it uses `divmod`. Can you replace the use of `time_from_micros` with this?

Reply:
Sure thing, updated 👍🏻